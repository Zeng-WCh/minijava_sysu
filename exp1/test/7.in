9+31+(27-25+28-16+33)*1+2+39-(10/26*27-20+30)*19/18+(39+28*10)*6*9+(16+28+8-27-3/39*25/14-33-34)-16+(7+25*25/7-19+25/39)/4*6+35-(26-34-15/18+38*15)*27-(14/1*25+4-2)/7-(32+30-20)/15*(24+15*5-23*4+20*9*20+1/9)+21/27*(25/23+32*26+19/8+11*4)*14-38/13-(37+32*35/38/6*30*6+16)-26/(33-13/13-31/26*35)+23/(17/35*8-28*7/31-29*7+27/28)+29+(31*9+29*23*37)*12/(35*18+40+21-12/7+35+20/20)/12*(14*19-5/38/5)+31*(35+3/11*18)*35/(31+5+3-7+28/25-36/34+25-34)*37*(1-30+36*6+40/24+24-25-17+20)*26/13-(18*26+30/14*20/19+36*16/36)-1-(25-37-37)*25*(2-38+23-36)/3+(6+3/19)*31+2+(5-38-33)/7/(30*3/30-32/28-30*14)+40-(39-2+7/17*2-17-12/27)+2*(9*23/20/28/23/30)*17-(40-1-1+21+18+30+14/24-36)*30+3-(3/9/22/17/25/6*31*5)+33*(23/9*31+4-20)/32-(35*24-7/10-24)/5+35-3*(9*34-31*8)/14+27*(33+1/24-9*32-8*20)+28*(7-12-40)-31-(17*23-29-16*34-8)-40*35/(26+1/25+32-3-1/34)+31*39/5/8/(26-10/14+4+23+21/20/11*30*9)+35/36-(20/5+40+20+25*10+4*10*9*24)+39-(29/12*38+17+24-17*30-21/17)*3+(39/16+18)-37*18+20-(40*34+12*31/7*15/18*5*39)*21*30-22*(39-13-28)*8*(6-13/32-11+30-18-21)-32-(39-14+26-39-39)+3-34-(10*10-31/1-24+18+36*7*40+8)/11+34-(13+10-12)*37/(27/3+14/16/1+24/19+13*17*29)+6-(23-22*24+1/4*12*10+13*39)-8+(4-16*8-23)+25/(13-1/2*20-34-19*14-20/2)+22*(28/15-12*23/22/11+19*2*8-14)-2*15-(15/14-16*27+31)*2*7/(15+20-31/4+22)-14*(31/37*25)+12+(16*37-35+13/7*9-9*19+37)-24/32-35-(18+21/33*27-2-22*18/16/38+5)/11*(12/38-5/39-8*30+26)*14/(40/37-6+24/14+4+17+22-3/3)/28*38/35*10*(33/2*32+21*22+9+35)+37-(20-40-31*27/2-18-2+19)/15-(6*32-39/5+10-7/16+16+36+23)/28+(12/5*21)+16/5*6*(1+18/28-7+3-32)-15