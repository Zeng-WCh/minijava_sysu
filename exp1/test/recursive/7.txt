6+7+0-6+6+3+6+8+2+4+3+9+1+3-0+1+6-9+5+0-9+9-7+9+8+8+4-4-1+0