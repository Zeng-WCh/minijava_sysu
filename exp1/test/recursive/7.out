Input an infix expression and output its postfix notation:
36+1-3+2+8-3+9+7-6+2+7+6+9-7+0+4+9+3-2-4-5-0-8+5-8-8-7-8-6-
End of program.
